Assets/Scripts/Input/InputRouter.cs

```diff
@@ -27,6 +27,8 @@ public class InputRouter : Singleton<InputRouter>
 
     public bool HandsVisible { get; private set; }
 
+    public Vector2 xamlMousePosition = new Vector2(0,0);
```

Comment:
This should probably be XamlMousePosition to match the casing of other public fields/properties.
There should also be a [HideInInspector()] attribute attached to it so it doesn't show up in Unity's inspector panel.